src/components/templates/DropDownPanel/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { TouchableOpacity, View, ViewStyle } from 'react-native';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming
+} from 'react-native-reanimated';
+import { Row, Spacer } from '@components/base';
+import { scale } from '@utils/scaling';
+import { COLORS } from '@constants/colors';
+import { BackIcon } from '@components/svg/icons';
+import { LayoutChangeEvent } from 'react-native/Libraries/Types/CoreEventTypes';
```

Comment:
wrong import of LayoutChangeEvent. It might be imported directly from 'react-native'